dashboard/app/controllers/activities_controller.rb

```diff
@@ -15,52 +15,92 @@ class ActivitiesController < ApplicationController
   MIN_LINES_OF_CODE = 0
   MAX_LINES_OF_CODE = 1000
 
+  # Records level progress after each time an attempt is made
+  #
+  # HTTP METHOD: POST only
+  #
+  # GET Params
```

Comment:
Yes, YARD is our recently adopted standard.  We use it in a few places already (e.g. net_sim_api.rb)